azure-resource-optimization-toolkit/README.md

```diff
@@ -0,0 +1,234 @@
+# **Azure Resource Optimization(ARO) Toolkit**
+Deploy an Azure Automation account with preconfigured runbooks and schedules to your subscription and start saving money.  Azure resource optimization happens automatically on your subscription once you enable or create your own schedule including on new resources!  
+
+***Objective:*** Provide decentralized automation capabilities for customers who want to reduce their costs.  Features include: 
+1.  Auto Snooze ARM VMs based on low CPU
+
+2.  Schedule ARM VMs to snooze/un-snooze
+
+3.  Schedule ARM VMs to snooze/un-snooze in ascending and descending order using Azure Tags
+
+4.  Bulk delete resource groups on demand
+
+# **How to On-board - ARO Toolkit**
+
+# **Step 1 Deploy ARO Toolkit**
+
+## **From Github**
+Click Deploy to Azure and proceed to Step 2, "Configure Azure Automation Variables".
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2FMSITARM%2Fmaster%2Fazure-resource-optimization-toolkit%2Fazuredeploy.json)
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2FMSITARM%2Fmaster%2Fazure-resource-optimization-toolkit%2Fazuredeploy.json" target="_blank">
+<img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+Important!  A bootstrap runbook can take up to 15 minutes to complete setting up your Azure Automation account.  Before you use ARO Toolkit, validate the Azure Run As account is created as show below.  If not, follow the steps in Appendix D. 
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/azure-resource-optimization-toolkit/images/AROToolkit/validateRunAs.png">
+    
+
+# **Step 2 Configure Azure Automation Variables**
+
+1.  Navigate to Azure portal and select your subscription (if you have
+    more than one)
+
+2.  Click on “**Automation Accounts**” from the service blade
+
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/azure-resource-optimization-toolkit/images/AROToolkit/image10.png">
+    
+
+3.  select the Azure Automation Account which you deployed
+
+4.  Navigate to the dashboard by clicking “Overview”
+
+5.  Click “**Assets**”
+
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/azure-resource-optimization-toolkit/images/AROToolkit/image11.png">
+    
+
+6.  Click “**Variables**” tile
+
+    <img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/azure-resource-optimization-toolkit/images/AROToolkit/image12.png">
+    
+
+7.  If needed you can maximize the variables window to get better
+    visibility
+
+8.  Update the variables as per your requirements
+
+    **Note:** If you want to act on all the VM’s at subscription level,
+    then you don’t need to specify the below \#a step.
```

Comment:
It means if you want to have all the VMs in the subscription acted on then leave the External\_ResourceGroupNames.  I updated this section to be more clear.